src/__tests__/core/web_api/legacy_api.test.js

```diff
@@ -40,6 +40,26 @@ describe('Auth0LegacyAPIClient', () => {
       const { overrides } = mock.WebAuth.mock.calls[0][0];
       expect(overrides).toEqual({ __tenant: 'tenant1', __token_issuer: 'issuer1' });
     });
+    it('forwards options to WebAuth', () => {
+      const options = {
+        redirectUrl: '//bar',
+        responseMode: 'query',
+        responseType: 'code',
+        plugins: [{ name: 'Plugin' }]
+      };
+      const callMapping = {
+        redirectUrl: 'redirectUri',
+        responseMode: 'responseMode',
+        responseType: 'responseType',
+        plugins: 'plugins'
+      };
+      const client = getClient(options);
+      const mock = getAuth0ClientMock();
+      const calledOptions = mock.WebAuth.mock.calls[0][0];
```

Comment:
use the snapshot testing here, since it's easier to maintain and read what's going on:
`expect(calledOptions).toMatchSnapshot();` and run the jest tests.